### DISCUSSION TITLE
Grouping Sounds

### DISCUSSION THREAD

**Discussion by ProphetofTruth-7** [ScratchAddons/ScratchAddons#5417] [Ideas]
As the title suggests, I was wondering if a better way to move sounds between sprites could be made. 
Grouping sounds in a folder and moving the folder only moves the top sound in the folder.

**Comment by ProphetofTruth-7** [comment]
Follow up question, is it even possible for variables and lists to react to the contents of the sound tab
(Like easily putting the names of the sounds in a list without having to do it manually)

**Reply by BroJac5246** [reply]
I don't think so, although one could theoretically write a program (in a different programming language) that automatically does so. That's probably not something that would be added to Scratch Addons, though.
